extension/llm/export/quantizer_lib.py

```diff
@@ -177,6 +177,9 @@ def get_qnn_quantizer(
         quant_dtype = QuantDtype.use_8a8w  # pyre-fixme[16]
     elif quant_config == "16a16w":
         quant_dtype = QuantDtype.use_16a16w  # pyre-fixme[16]
+        # Due to the error with 16a16w in Qnn Htp, we need to disable per channel linear quantization when use 16a16w
+        # TODO: enable it after the issue is fixed
+        qnn_quantizer.set_per_channel_linear_quant(enable=False)
```

Comment:
Maybe add a warning just in case. Users probably will expect better accuracy with 16a16w, and disable per channel can regress the accuracy.

Reply:
Got it. Let me add a warning here
```
logging.warn("Disable per channel linear quantization due to the error with QNN HTP 16a16w.")
```